modules/virtualnetwork/locals.tf

```diff
@@ -28,6 +36,8 @@ locals {
         this_resource_id   = v.hub_network_resource_id
         remote_resource_id = azapi_resource.vnet[k].id
       }
+      peering_direction   = contains(local.valid_peering_directions, coalesce(lower(v.hub_peering_direction), local.peering_direction_both)) ? coalesce(lower(v.hub_peering_direction), local.peering_direction_both) : local.peering_direction_both
```

Comment:
What are your thoughts on using a variable validation rule here, with for loop and alltrue() Could we simplify this?

Reply:
Ok fair, happy with this